### DISCUSSION TITLE
Using maftools to visualize germline variants

### DISCUSSION THREAD

**Discussion by whitejaa** [PoisonAlien/maftools#762] [Q&A]
Hi,

This isn't so much an issue as it is a general question. I would like to visualize and analyze germline variants using maftools, however I am concerned that it may not be the right tool for that. I couldn't find a pubmed article referencing the use of maftools to do this and the maftools pub does not mention germline variants. Is my concern valid? If so, do you know of any tools that would be comparable to maftools but could be used for germline mutations?

Thanks!

**Comment by oalavijeh** [comment]
Hey I had the same query (see #751 ) and got it to work. 

Otherwise the lollipop tool is great (https://bioconductor.org/packages/release/bioc/vignettes/trackViewer/inst/doc/lollipopPlot.html)

Hope that's helpful!

**Comment by whitejaa** [comment]
Thanks for the help!

**Comment by PoisonAlien** [comment]
Hi,
You can use it for any kind of data as long as it resembles MAF format.
